src/status_im2/contexts/chat/home/style.cljs

```diff
@@ -22,3 +22,16 @@
    :right       0
    :left        0
    :padding-top top})
+
+(def header-height 245)
```

Comment:
Header height was a number in the view namespace, since I will use it many times, I added it to the styles namespace and referenced it